### PR TITLE
Bug fixes based on user feedback

### PR DESCRIPTION
**Pull request by xiangshen-dk**
Updating the configuration and doc.

Internal bug ids:
156582220, 156573266, 156572180, 156583979

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,26 +56,52 @@ Follow these steps to configure a service account for the backend application:
...
-1. Select the **Service accounts** tab.
-1. Click **Manage all service accounts**.
+1. Click the **Settings** gear next to 'Project Overview' and then **Project settings**.
+1. Select **Service accounts** and click the link **Manage service account permissions**.
 1. In the **IAM & admin** page click **Create service account**.
 1. In the dialog, create an account with the following parameters:
    * Enter *playchat-servlet* in the **Service account name** field.
    * Select **Project** > **Owner** in the **Role** menu.
-   * Select **Furnish a new private key**.
+   * Check **CREATE KEY**.
```
Comment (ricalo) [inline README.md:65]: Is **Create key** a checkbox? If so, use select instead. Per https://developers.google.com/style/word-list#check

Ignore otherwise.

File: pom.xml [hunk 2]
```diff
@@ -1,95 +1,112 @@
...
-  <groupId>com.google.cloud.solutions.flexenv</groupId>
-  <artifactId>backend</artifactId>
-  <packaging>war</packaging>
-  <version>0.0.1-SNAPSHOT</version>
-  <name>App Engine Backend module for Firebase</name>
-  <url>https://cloud.google.com</url>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```
Comment (ricalo) [inline pom.xml:4]: What is the reason for the change in indentation?

File: src/main/webapp/Dockerfiles [hunk 3]
```diff
@@ -1,2 +0,0 @@
-FROM gcr.io/google_appengine/jetty9-compat:githubheadasync
```
Comment (ricalo) [inline src/main/webapp/Dockerfiles:1]: I don't understand the rationale behind the changes in this file. Could you provide some context?

File: README.md (hunk 1 above)
Comment (xiangshen-dk) [reply README.md]: No, it's not a checkbox. It's a button you can click.

File: pom.xml (hunk 2 above)
Comment (xiangshen-dk) [reply pom.xml:4]: That's reformatted by my IDE so they have consistent indentations with the rest of the file.

File: src/main/webapp/Dockerfiles (hunk 3 above)
Comment (xiangshen-dk) [reply src/main/webapp/Dockerfiles:1]: It was used to customize the environment. But it's not needed anymore.

File: pom.xml (hunk 2 above)
Comment (ricalo) [reply pom.xml:4]: Mmm... changes should not be the result of your IDE configuration. It makes it difficult to identify the changes that you are actually proposing. Additionally, the next person to propose a change might have a different configuration.
May I suggest that you configure your IDE with 2-space indentation, per https://google.github.io/styleguide/xmlstyle.html?

**Comment by xiangshen-dk** [comment]
Made updates.

**Comment by xiangshen-dk** [comment]
Sorry, clicked the wrong button.

**Comment by ricalo** [comment]
> Sorry, clicked the wrong button.

No problem. I just reopened one of the pending comments.

**Comment by xiangshen-dk** [comment]
Good point for the indentation. I have reformatted the pom.xml.

**Comment by ricalo** [comment]
> Good point for the indentation. I have reformatted the pom.xml.

Thank you. LGTM as soon as the tests pass.
